zebra/zebra_fpm_netlink.c

```diff
@@ -287,6 +287,8 @@ static int netlink_route_info_fill(struct netlink_route_info *ri, int cmd,
 
 		if (CHECK_FLAG(nexthop->flags, NEXTHOP_FLAG_RECURSIVE))
 			continue;
+		if (CHECK_FLAG(nexthop->flags, NEXTHOP_FLAG_DUPLICATE))
```

Comment:
Maybe we should use `NEXTHOP_IS_ACTIVE()` here? It also would check against `NEXTHOP_FLAG_ACTIVE`.

Reply:
Hi, In line 309, before netlink_route_info_add_nh, it will do CHECK_FLAG(nexthop->flags, NEXTHOP_FLAG_ACTIVE). So I don't think we need to check NEXTHOP_IS_ACTIVE() here, it will do NEXTHOP_FLAG_ACTIVE check repeatedly.